platforms/gke/base/core/teardown.sh

```diff
@@ -17,9 +17,7 @@ set -o errexit
 
 start_timestamp=$(date +%s)
 
-terraservices=("workloads/kueue" "gke_enterprise/fleet_membership" "container_node_pool" "container_cluster" "networking")
-# Disable gke_enterprise/servicemesh due to b/376312292
-#terraservices=("workloads/kueue" "gke_enterprise/servicemesh" "gke_enterprise/fleet_membership" "container_node_pool" "container_cluster" "networking")
+declare -a terraservices=${TERRASERVICES:-("networking" "container_cluster" "container_node_pool" "gke_enterprise/fleet_membership" "workloads/kueue")}
```

Comment:
Considering that we're repeating this definition and it's part of the core platform, consider moving this definition in `platforms/gke/base/_shared_config/scripts/set_environment_variables.sh`

Reply:
These scripts are just meant to be used for testing and reference, each use case would implement their logic as needed. `set_environment_variables.sh` takes the `variables` and `local` in Terraform and sets them as environment variables for use in other tools.